tests/test_dask_image/test_ndinterp/test_affine_transformation.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import pytest
+
+import dask_image.ndinterp as da_ndinterp
+
+import numpy as np
+import dask.array as da
+
+
+@pytest.mark.parametrize("n", [1, 2, 3, 4])
+@pytest.mark.parametrize("interp_order", [0, 1, 3])
+def test_affine_transformation(n, interp_order):
+    """
+    Compare the outputs of `ndimage.affine_transformation`
+    and `dask_image.ndinterp.affine_transformation`.
+
+    Notes
+    -----
+        Currently, prefilter is disabled and therefore the output
+        of `dask_image.ndinterp.affine_transformation` is compared
+        to `prefilter=False`.
+    """
+
+    # define test image
+    a = 25
+    im = np.random.random([a] * n)
```

Comment:
Suggestion purely for readability & consistency throughout dask-image: let's use `image` instead of the abbreviation `im`.

Reply:
Done